docs/OTHER.md

```diff
@@ -14,11 +14,21 @@ This sample highlights copying Data Views, Assets, and Streams from a source Nam
-| ------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| [UOM](https://github.com/osisoft/sample-ocs-uom-dotnet)                                           | [.Net](https://github.com/osisoft/sample-ocs-uom-dotnet)                                                                                          | [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OCS/osisoft.sample-ocs-uom-dotnet?repoName=osisoft%2Fsample-ocs-uom-dotnet&branchName=main)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=2625&repoName=osisoft%2Fsample-ocs-uom-dotnet&branchName=main)                                                 |
-| [Namespace Data Copy](https://github.com/osisoft/sample-ocs-namespace_data_copy-python)           | [Python](https://github.com/osisoft/sample-ocs-namespace_data_copy-python)                                                                        | [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OCS/osisoft.sample-ocs-namespace_data_copy-python?repoName=osisoft%2Fsample-ocs-namespace_data_copy-python&branchName=main)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=3856&repoName=osisoft%2Fsample-ocs-namespace_data_copy-python&branchName=main) |
-| [Sample Libraries](https://github.com/osisoft/OSI-Samples-OCS/blob/main/docs/SAMPLE_LIBRARIES.md) | [Java](https://github.com/osisoft/sample-ocs-sample_libraries-java) </br> [Python](https://github.com/osisoft/sample-ocs-sample_libraries-python) | ~~~~~~~~~~~~~~~~~~~ </br> ~~~~~~~~~~~~~~~~~~~                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
Sorry, My first attempt was not rendering correctly, and I had to completely restructure this table before I could find my typo. So this whole chunk is marked as changed, but it should just be whitespace removed